<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>easyPOS - Касса</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="centralLayout">
    <item>
     <widget class="QWidget" name="centralPanel" native="true">
       <layout class="QVBoxLayout" name="rightLayout">
        <item>
         <widget class="QLabel" name="checkLabel">
          <property name="text">
           <string>Чек</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="checkWidget">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <column>
           <property name="text">
            <string>Товар</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Кол-во</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цена</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Сумма</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="totalsLayout">
          <item>
           <widget class="QLabel" name="totalLabel">
            <property name="text">
             <string>Итого: 0,00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="discountLabel">
            <property name="text">
             <string>Скидка: 0,00</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="toPayLabel">
            <property name="text">
             <string>К оплате: 0,00</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="cartButtonsLayout">
          <item>
           <widget class="QLabel" name="qtyLabel">
            <property name="text">
             <string>Кол-во:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="qtySpinBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="findGoodButton">
            <property name="text">
             <string>Поиск</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newCheckButton">
            <property name="text">
             <string>Новый чек</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="removeFromCartButton">
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="discountButton">
            <property name="text">
             <string>Скидка</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="payButton">
            <property name="text">
             <string>Оплатить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справочники</string>
    </property>
    <addaction name="actionCategories"/>
    <addaction name="actionGoods"/>
    <addaction name="actionServices"/>
    <addaction name="actionVatRates"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Помощь</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCategories">
   <property name="text">
    <string>Категории товаров</string>
   </property>
  </action>
  <action name="actionGoods">
   <property name="text">
    <string>Товары</string>
   </property>
  </action>
  <action name="actionServices">
   <property name="text">
    <string>Услуги</string>
   </property>
  </action>
  <action name="actionVatRates">
   <property name="text">
    <string>Ставки НДС</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
